src/sdk/conference/signaling.js

```diff
@@ -149,4 +151,43 @@ export class SioSignaling extends EventModule.EventDispatcher {
       });
     });
   }
+
+  /**
+   * @function _onReconnectionTicket
+   * @instance
+   * @desc Parse reconnection ticket and schedule ticket refreshing.
+   * @memberof Oms.Conference.SioSignaling
+   * @private.
+   */
+  _onReconnectionTicket(ticketString) {
+    this._reconnectionTicket = ticketString;
+    const ticket = JSON.parse(Base64.decodeBase64(ticketString));
+    // Refresh ticket 1 min or 10 seconds before it expires.
+    const now = Date.now();
+    const millisecondsInOneMinute = 60 * 1000;
+    const millisecondsInTenSeconds = 10 * 1000;
+    if (ticket.notAfter <= now - millisecondsInTenSeconds) {
```

Comment:
I think in this situation you still need to refresh?

Reply:
If the ticket is only valid for less than 10s, it's supposed to be an incorrect configuration because of following two reasons:
- Frequently refresh reconnection ticket consumes unnecessary network bandwidth and server resources.
- The ticket is likely to be invalid when you reconnected to server.